tests/conftest.py

```diff
@@ -461,6 +490,28 @@ def table_update(req: tsi.TableUpdateReq) -> tsi.TableUpdateRes:
         )
         return client.server.table_update(req)
 
+    @app.post("/feedback/create")
+    def feedback_create(req: tsi.FeedbackCreateReq) -> tsi.FeedbackCreateRes:
+        records.append(
+            (
+                "feedback_create",
+                req,
+            )
+        )
+        return client.server.feedback_create(req)
+
+    @app.post("/feedback/create_batch")
```

Comment:
Similarly, the canonical format for batch APIs are
```
POST /feedback/batch/create
```

This is less strongly specified than the single-item endpoints format but batch APIs usually simply put a batch prefix or suffix.